coinstacks/bitcoin/pulumi/Pulumi.sample.yaml

```diff
@@ -33,6 +33,13 @@ config:
       replicaCount: 1
       storageSize: '1Gi'
 
+    ## rabbit - if specified a rabbit cluster will be deployed
+    rabbit:
+      cpuLimit: '500m'
+      memoryLimit: '512Mi'
+      replicaCount: 1
+      storageSize: '5Gi'
```

Comment:
Sample defaults are meant to be minimal case for local kube dev. Is `5Gi` required by cluster or can we go with `1Gi`?